src/test/libcephfs/test.cc

```diff
@@ -36,6 +36,10 @@
 #include <vector>
 #include <thread>
 
+#ifndef ALLPERMS
+#define ALLPERMS (S_ISUID|S_ISGID|S_ISVTX|S_IRWXU|S_IRWXG|S_IRWXO)
```

Comment:
to build under alpine, `ACCESSPERMS` should be defined in `common_init.cc` too

Reply:
why? this macro is not referenced out there, i think.